### ISSUE TITLE
Replacements in title text don't work

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#185]
I'm not sure how much replacements should work in the title field, but...

The following file has problems:

```
.. _label_in_other_file:

some X title |XXX|
==================

:ref:`label_in_other_file`.


.. role:: red-html(raw)
   :format: html

.. |XXX| replace:: :red-html:`<font color="#882222">xxx</font>`
```

So the label |XXX| is to be replaced with a raw output (in HTML). In the title itself, this works fine. In the reference to the title, it prints it out "<font ....>". Note that this is not a problem if the replacement text is `**xxx**`, which leads me to think its being deliberately stripped, but that replacements arent handled in this case.

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/185
- Originally reported by: Paul Biggar
- Originally created at: 2009-05-28T12:40:02.807

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2013-03-31 13:57:12+00:00_

Removing version: 0.6 (automated comment)

**Comment by tk0miya** [comment]
Fixed at 3b9aee4.
